oneanddone/users/views.py

```diff
@@ -151,6 +151,15 @@ def login_failure(self, *args, **kwargs):
             logging in, let us know by emailing <a href="mailto:{email}">{email}</a>.
         """).format(email='oneanddone@mozilla.com'), extra_tags='safe')
         return super(Verify, self).login_failure(*args, **kwargs)
+    
+    def login_success(self, *args, **kwargs):
+        if (not UserProfile.objects.filter(user=self.request.user).exists()):
```

Comment:
@tessie There is a problem here because `self.request.user` has not been set to the logged in user yet, so this statement is generating an error. You need to use `self.user` instead. That will fix this issue, but I have come across another one that I am asking the developer of django-browserid for help with. I will update this when I have a solution.